sshnp_docker_demo/demo-base/Dockerfile

```diff
@@ -0,0 +1,48 @@
+FROM debian:stable-20230522-slim@sha256:d828cca5497a2519da9c6d42372066895fa28a69f1e8a46a38ce8f750bd2adf0 AS buildimage
+
+WORKDIR /app
+
+RUN set -eux ; \
+    apt-get update ; \
+    apt-get install -y openssh-server wget sudo iputils-ping iproute2 ncat \
+    telnet net-tools nmap iperf3 tmux traceroute nano vim watch curl dpkg ; \
+    case "$(dpkg --print-architecture)" in \
+    amd64) \
+    ARCH="x64";; \
+    armhf) \
+    ARCH="arm";; \
+    arm64) \
+    ARCH="arm64";; \
+    riscv64) \
+    ARCH="riscv64";; \
+    esac; \
+    URL="https://api.github.com/repos/atsign-foundation/sshnoports/releases/latest" ; \
+    VERSION=$(curl --silent ${URL} | grep -Po '"tag_name": "v\K.*?(?=")') ; \
+    URLP="https://github.com/atsign-foundation/sshnoports/releases/download" ; \
+    wget "${URLP}/v${VERSION}/sshnp-linux-${ARCH}.tgz" ; \
+    tar -xvf sshnp-linux-${ARCH}.tgz ; \
+    rm sshnp-linux-${ARCH}.tgz ;
+
+FROM ubuntu:22.04@sha256:dfd64a3b4296d8c9b62aa3309984f8620b98d87e47492599ee20739e8eb54fbf
```

Comment:
This shouldn't need to be a two stage build. If we're using something as heavy as Ubuntu, then it should be possible to just do everything in place.

Reply:
https://github.com/atsign-foundation/at_demos/pull/97/commits/ba2205149939f2658e5785af5da51c42e02d8599